flask_ipblock/__init__.py

```diff
@@ -59,7 +69,10 @@ def block_before(self):
         ip = ip.rsplit(',', 1)[-1].strip()
 
         if self.matches_ip(ip):
-            return 'IP Blocked', 200
+            if self.logger is not None:
+                self.logger.info("IPBlock: matched {0}, {1}".format(ip, self.block_msg))
```

Comment:
No need for `0` and `1` in `{}` if the order of the args matches the order of the braces.